### PR TITLE
Fyber - 7.8.5.0

### PR DESCRIPTION
**Pull request by TomerFyber**

### CODE REVIEW

File: Fyber/FyberAdapterConfiguration.m [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- (NSString *)biddingToken {
+    return nil;
+}
+
+- (NSString *)moPubNetworkName {
+    return @"fyber";
+}
+
+- (NSString *)networkSdkVersion {
+    return IASDKCore.sharedInstance.version;
```
Comment (dravyan) [inline Fyber/FyberAdapterConfiguration.m:56]: The adapter version is always 1 decimal greater than the SDK version. In this case, this should be `7.8.2.0`

Comment (TomerFyber) [reply Fyber/FyberAdapterConfiguration.m:56]: Should we change it here? Or in `- (NSString *)adapterVersion`? (or both?)

File: Fyber/FyberRewardedVideoCustomEvent.m [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    }
+}
+
+- (void)IAAdWillLogImpression:(IAUnitController * _Nullable)unitController {
+    MPLogAdEvent([MPLogEvent adShowSuccessForAdapter:NSStringFromClass(self.class)], self.spotID);
+    [self.delegate fullscreenAdAdapterDidTrackImpression:self]; // manual track;
+}
+
+// in order to use the rewarded callback for all available rewarded content, you will have to implement this method (not the `IAVideoCompleted:`;
+- (void)IAAdDidReward:(IAUnitController * _Nullable)unitController {
```
Comment (dravyan) [inline Fyber/FyberRewardedVideoCustomEvent.m:215]: This callback is not fired for rewarded video completion. Instead, only `IAVideoCompleted` is fired. Is that expected?

File: Fyber/FyberAdapterConfiguration.m [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            } else {
+                IASDKCore.sharedInstance.GDPRConsent = IAGDPRConsentTypeGiven;
+            }
+        } else {
+            const BOOL canCollectPersonalInfo = MoPub.sharedInstance.canCollectPersonalInfo;
+            
+            IASDKCore.sharedInstance.GDPRConsent = (canCollectPersonalInfo) ? IAGDPRConsentTypeGiven : IAGDPRConsentTypeDenied;
+        }
+        
+        [NSUserDefaults.standardUserDefaults setBool:YES forKey:kIASDKShouldUseMopubGDPRConsentKey];
```
Comment (dravyan) [inline Fyber/FyberAdapterConfiguration.m:138]: `[NSUserDefaults.standardUserDefaults setBool:YES forKey:kIASDKShouldUseMopubGDPRConsentKey];`
trying to set the user value here like consent. Is that the intention? how is `forKey:kIASDKShouldUseMopubGDPRConsentKey` value set in the app?. It is recommended not to pass this information via adapters. Can you handle it in Fyber SDK?

File: Fyber/FyberAdapterConfiguration.h [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#import <Foundation/Foundation.h>
+#if __has_include(<MoPub/MoPub.h>)
+#import <MoPub/MoPub.h>
+#elif __has_include(<MoPubSDK/MoPub.h>)
+#import <MoPubSDK/MoPub.h>
+#else
+#import <MoPub.h>
```
Comment (TomerFyber) [inline Fyber/FyberAdapterConfiguration.h:7]: @dravyan `#import <MoPub.h>` Seems to be needed here in order to compile.

**Pull request merged**